career/expectations/software-development-analyst/README.md

```diff
@@ -0,0 +1,130 @@
+- **Often reviews other people's codes via a code review process**. Is capable of pointing out problems or improvements.
+- Asks for help whenever they are feeling blocked, not productive or doesn't know what to do.
+- Usually codes "by example" (e.g. copying code), **but knows what they are doing and is usually capable of learning on the fly**.
+- Knows a bit about continuous delivery and why that's important, but **often creates large PRs or pieces of code that are hard to review and ship efficiently**.
+- Is proactive about starting new tasks. Usually does not need someone else to tell them what to do, as long as there are tasks that conform to our DoR (Definition of Ready).
+
+### Hard Skills
+
+- **Is experienced in most of the tech stack that we use** (Ruby on Rails, React, Next.js, TypeScript, Python, GraphQL and others). May have occasional blocks, but is able to develop efficiently.
+- **Is capable of using TDD daily, and understands the importance of it**.
+- Is capable of writing code according to our [Styleguide](https://github.com/GeekHunter-Brasil/hello-world).
+- Has experience with common database technologies. Knows relational models, tables, views, indices, and other terms. Is capable of writing a performant SQL query.
+- Has experience in git, and **is able to deal with merge conflicts**.
+- Is fluent in Object Oriented Programming, knows the key points to it and is able to write high-quality OOP code.
+- Knows what is Functional Programming, knows advantages and disadvantages to each paradigm.
+- Understands concepts such as data immutability, pure functions, and reactivity.
+- **Understands concepts about software architecture, and knows how to follow common patterns** such as Clean Architecture, Hexagonal Architecture and others. Knows about their advantages and disadvantages.
+- Understands how to design applications and is capable of choosing a good design according to the complexity or challenge (e.g. monoliths, micro-services, serveless, etc).
+- **Knows how to debug code**. Is capable of debugging their own code, manipulating the memory and instructions to reproduce a specific context.
+- Usually is capable of debugging external code (such as a dependency). For example, is capable of creating a fork + pull request in a dependency in case something does not work.
+- Is capable of solving any problem that doesn't need R&D (Research & Development) efficiently.
+
+### Soft Skills
+
+- Is able to keep their presence in synchronous (e.g. squad rituals such as dailies, plannings, reviews; pair programmings) and asynchronous moments (e.g. discussions on Slack or Basecamp).
+- Is really motivated to learn more and develop their skills.
+- Usually understands their own strong or weak points, and sees something they want to prioritize in their career. The leadership process here is usually easier, since this person is able to develop their skills with some level of autonomy.
```

Comment:
```suggestion
- Usually understands their own strengths and weaknesses, and sees something they want to prioritize in their career. The leadership process here is usually easier, since this person is able to develop their skills with some level of autonomy.
```